test/dotnet-test.UnitTests/GivenATestCommand.cs

```diff
@@ -14,10 +14,13 @@ namespace Microsoft.Dotnet.Tools.Test.Tests
     public class GivenATestCommand
     {
         private static readonly string ProjectJsonPath = Path.Combine(
-                AppContext.BaseDirectory,
+                Directory.GetCurrentDirectory(),
```

Comment:
OK - that sounds good to me. Just making sure this was consistent with the other tests that get TestAssets.